src/emulator/storage/apis/request.ts

```diff
@@ -0,0 +1,18 @@
+import { Request } from "express";
```

Comment:
Would put this file further up (maybe in `src/emulator`?) since it's not specific to Storage

Reply:
done